codespell_lib/data/dictionary.txt

```diff
@@ -37655,6 +37663,8 @@ parallellized->parallelized
 parallellizes->parallelizes
 parallellizing->parallelizing
 parallells->parallels
+parallelpiped->parallepiped
```

Comment:
Correct spelling is `parallelepiped`.

Reply:
Oops. I thought that's what I had typed. Maybe falsely auto-corrected.

Reply:
I wonder why tests didn't catch that.

Reply:
I have fixed it and will merge.